src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../Button.css';
+import PropTypes from 'prop-types';
+
+const isEqual = val => val === '=';
+
+const Button = ({ name }) => (
+  <button type="button" className={isEqual(name) ? 'eqlBtn' : 'btn'}>
+    { name }
+  </button>
+);
+
+Button.propTypes = {
+  name: PropTypes.node.isRequired,
```

Comment:
- [ ] 'name' should have a prop of the type string, not a node. Kindly update the proptype to string.